### DISCUSSION TITLE
How to change/remove/add health in health bar?(MODCHART)

### DISCUSSION THREAD

**Discussion by PlayerPro342** [KadeArchive/Kade-Engine#693] [Q&A] [answered]
I always have fun with modcharts...
I've been seeing many modcharts that have new stuff like
the notes separate in vs tree.
But never seen a modchart change the health or health bar.

**Comment by puyoxyz** [comment, marked as answer]
https://github.com/KadeDev/Kade-Engine/pull/582

**Comment by PlayerPro342** [comment]
Thanks, setHealth worked.
